### PR TITLE
CVS-79977 Fix model ensemble example and move to demos directory

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: demos/model_ensemble/python/README.md [hunk 1]
```diff
@@ -31,34 +31,34 @@ This guide shows how to implement a model ensemble using the [DAG Scheduler](dag
...
+~$ docker run -u $(id -u):$(id -g) -v ~/models:/models:rw openvino/ubuntu18_dev:latest deployment_tools/model_optimizer/mo_tf.py --input input1,input2 --input_shape '[1,1001],[1,1001]' --saved_model_dir /models/public/argmax/saved_model/ --output_dir /models/public/argmax/1
 ~$ mkdir -p ovms_models/googlenet-v2-tf/1
 ~$ mkdir -p ovms_models/resnet-50-tf/1
 ~$ mkdir -p ovms_models/argmax/1
 ~$ cp ~/models/public/googlenet-v2-tf/FP32/googlenet-v2-tf.{bin,xml} ~/ovms_models/googlenet-v2-tf/1/
 ~$ cp ~/models/public/resnet-50-tf/FP32/resnet-50-tf.{bin,xml} ~/ovms_models/resnet-50-tf/1/
 ~$ cp ~/models/public/argmax/1/saved_model.{bin,xml} ~/ovms_models/argmax/1/
 
-~$ tree models/public
+~$ tree ovms_models
```
Comment (mzegla) [inline demos/model_ensemble/python/README.md:48]: Is it possible to prepare such directory with a Makefile? And also call it `workspace` to match the convention in other demos?

File: demos/README.md [hunk 2]
```diff
@@ -31,7 +32,8 @@ OpenVINO Model Server demos have been created to showcase the usage of the model
...
+|[Multi Faces Analysis Pipeline](multi_faces_analysis_pipeline/python/README.md)|Run prediction on a JPEG image using a pipeline of age-gender recognition and emotion recogition models via gRPC API to extract multiple faces from the image and analyze all of them. This demo uses [pipeline](../docs/dag_scheduler.md) with [model_zoo_intel_object_detection custom node](https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/model_zoo_intel_object_detection) and [demultiplexer](../docs/demultiplexing.md) |
+|[Model Ensemble Pipeline](model_ensemble/python/README.md)|Combine multiple models into one pipeline and aggregate results before sending a response. This demo uses [pipeline](../docs/dag_scheduler.md). |
```
Comment (mzegla) [inline demos/README.md:36]: I would stick to the convention that we focus on the use case in the main demo list. Let's at least mention that we do image classification in that demo:
```
"Combine multiple image classification models into one pipeline and aggregate results to improve classification accuracy"
```

What do you think?

File: demos/model_ensemble/python/README.md (hunk 1 above)
Comment (dkalinowski) [reply demos/model_ensemble/python/README.md]: done

File: demos/README.md (hunk 2 above)
Comment (dkalinowski) [reply demos/README.md]: done

File: demos/model_ensemble/python/README.md [hunk 3]
```diff
@@ -146,85 +126,95 @@ Use the config.json as given below
...
-~/model_server$ . .venv/bin/activate && cd client/python/tensorflow-serving-api/samples
+~$ cd model_server/client/python/tensorflow-serving-api/samples
+~/model_server/client/python/tensorflow-serving-api/samples$ virtualenv .venv
+~/model_server/client/python/tensorflow-serving-api/samples$ . .venv/bin/activate && pip3 install -r requirements.txt
 (.venv) ~/model_server/client/python/tensorflow-serving-api/samples$ python3 grpc_predict_resnet.py --pipeline_name image_classification_pipeline --images_numpy_path ../../imgs.npy \
     --labels_numpy_path ../../lbs.npy --grpc_port 9100 --input_name image --output_name label --transpose_input False --iterations 10
 Image data range: 0.0 : 255.0
 Start processing:
-        Model name: image_classification_pipeline
+        Model name: resnet
```
Comment (dtrawins) [inline demos/model_ensemble/python/README.md:149]: why the model name is resnet instread of image_classification_pipeline?

File: demos/model_ensemble/python/README.md [hunk 4]
```diff
@@ -1,72 +1,52 @@
...
+        └── resnet-50-tf.xml
+
+6 directories, 10 files
 ```
 
 ## Step 2: Define required models and pipeline <a name="define-models"></a>
 Pipelines need to be defined in the configuration file to use them. The same configuration file is used to define served models and served pipelines.
 
-Use the config.json as given below
+Use the config.json located in this directory, the content is as follows:
```
Comment (dtrawins) [inline demos/model_ensemble/python/README.md:47]: which is "this directory"?

**Review by dtrawins — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**